lxd/instance/drivers/driver_lxc.go

```diff
@@ -6179,7 +6182,7 @@ func (d *lxc) InsertSeccompUnixDevice(prefix string, m deviceConfig.Device, pid
 
 	// Bind-mount it into the container
 	defer os.Remove(devPath)
-	return d.insertMountLXD(devPath, tgtPath, "none", unix.MS_BIND, pid, false)
+	return d.insertMountLXD(devPath, tgtPath, "none", unix.MS_BIND, pid, "")
```

Comment:
@brauner should this be idmap.IdmapStorageNone or idmap.IdmapStorageUnknown rather than empty string for consistency?

Reply:
Yep, fixed.